lib/sgtn-client/api/translation.rb

```diff
@@ -11,25 +11,25 @@ module Core
   
   class Translation
 
-      def self.getString(component, key, locale)
-        SgtnClient.logger.debug "[Translation][getString_p]component=#{component}, key=#{key}, locale=#{locale}"
+      def self.getString(component, key, locale=SgtnClient::Config.configurations.default)
+        SgtnClient.logger.debug "[Translation][getString]component=#{component}, key=#{key}, locale=#{locale}"
         str = getTranslation(component, key, locale)
         if str.nil?
           str = SgtnClient::Source.getSource(component, key, SgtnClient::Config.configurations.default)
           if str.nil?
-            SgtnClient.logger.error "[Translation]can't find the key '" + key + "' in source path!"
+            SgtnClient.logger.debug "[Translation][getString]can't find key=#{key}, component=#{component}, locale=#{locale}!!!"
```

Comment:
This is for a different, higher-priority issue: https://github.com/vmware/singleton/issues/1493
Next time, please keep the PR as small as possible.